Src/LinearSolvers/MLMG/AMReX_MLMG.cpp

```diff
@@ -1113,10 +1126,9 @@ MLMG::prepareForSolve (const Vector<MultiFab*>& a_sol, const Vector<MultiFab con
     sol_raii.resize(namrlevs);
     for (int alev = 0; alev < namrlevs; ++alev)
     {
-        if (a_sol[alev]->nGrow() == 1)
+        if (a_sol[alev]->nGrow() >= 1)
```

Comment:
The method I implemented requires 2 ghost nodes. However I split off a separate `CFStrategy` check and replaced `>=` with `==` so it will not alter existing behavior.